.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - '0.10'
-  - '0.12'
-  - '5'
+  - '6'
```

Comment:
I suggest LTS, and current which is currently 4, 6, 7. Node Sass will be shifting it's support this configuration later this year.